<ui version="4.0" >
 <class>Config</class>
 <widget class="QDialog" name="Config" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>317</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Qxygen Setup</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="3" column="1" >
    <widget class="QComboBox" name="tabDirection" >
     <item>
      <property name="text" >
       <string>Sunken</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Relief</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Raised</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0" >
    <widget class="QLabel" name="label_6" >
     <property name="text" >
      <string>Tabbar 3D appereance</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" >
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Pushbuttons</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="2" column="0" >
       <widget class="QCheckBox" name="inverseButtons" >
        <property name="text" >
         <string>Inverse</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QCheckBox" name="raisedButtons" >
        <property name="text" >
         <string>Raised</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QCheckBox" name="roundButtons" >
        <property name="text" >
         <string>Round</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Colors</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="2" column="1" >
       <widget class="QComboBox" name="cmTab" />
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="label_5" >
        <property name="text" >
         <string>Tabbar</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QComboBox" name="cmScroll" />
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_4" >
        <property name="text" >
         <string>Scroll Slider</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QComboBox" name="cmProgress" />
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>Progress</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1" colspan="2" >
    <widget class="QComboBox" name="acceleration" >
     <item>
      <property name="text" >
       <string>None (Tiling system)</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>XRender</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>OpenGL</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="1" colspan="2" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Hardware Acceleration</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QComboBox" name="bgMode" >
     <item>
      <property name="text" >
       <string>Plain</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Scanlines</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Gradient</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Complex</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Background Mode</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Config</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Config</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
